test/jdk/java/net/httpclient/HttpRequestBuilderTest.java

```diff
@@ -262,13 +266,11 @@ public static void main(String[] args) throws Exception {
         // verify that the default HEAD() method implementation in HttpRequest.Builder
         // interface works as expected
         HttpRequest defaultHeadReq = new NotOverriddenHEADImpl().HEAD().uri(TEST_URI).build();
-        String actualMethod = defaultHeadReq.method();
-        if (!actualMethod.equals("HEAD")) {
-            throw new AssertionError("failed: expected HEAD method but got method: " + actualMethod);
-        }
-        if (defaultHeadReq.bodyPublisher().isEmpty()) {
-            throw new AssertionError("failed: missing bodyPublisher on HEAD request");
-        }
+        assertEquals("HEAD", defaultHeadReq.method(), "Method");
+        assertEquals(defaultHeadReq.bodyPublisher().isEmpty(), false, "Body publisher absence");
```

Comment:
The params here appear inverted. The first param to the method says "expected", so I think it should be:
```
assertEquals(false, defaultHeadReq.bodyPublisher().isEmpty(), "Body publisher absence");
```

Reply:
Fixed in 9f7b054b0e96244aea374dbf2555f2a03f61cab0.